clang/test/CodeGenSYCL/anonymous_integration_footer2.cpp

```diff
@@ -13,7 +13,8 @@ using namespace cl;
 struct S1 {
   static constexpr sycl::specialization_id a{1};
 };
-// CHECK: __SYCL_INLINE_NAMESPACE(cl) {
+// CHECK: #include <CL/sycl/detail/defines_elementary.hpp>
```

Comment:
Same here.

Reply:
Fixed in e56ee417374054e0bcccc8574192cf51601722cf